### PR TITLE
Fixed color and padding for the icons in dropdown

### PR DESCRIPTION
**Pull request by SebastianPietruszka**

### CODE REVIEW

File: src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ class DropdownItem extends React.Component {
         )}
         title={title}
       >
-        {Icon && <Icon left outline />}
+        {Icon && <Icon className="DropdownItem__button-icon" left outline />}
```
Comment (renaudtertrais) [inline src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.js]: Change spacing in Icon with `left` or `right` prop

Comment (SebastianPietruszka) [reply src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.js]: Done

File: src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss [hunk 2]
```diff
@@ -13,6 +13,7 @@
   display: flex;
   align-items: center;
   color: $coolgrey-60;
+  fill: $coolgrey-60;
```
Comment (renaudtertrais) [inline src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss]: why ? normally Icons use font color

Comment (SebastianPietruszka) [reply src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss]: It's change for circle in radio button icon. Font color doesn't help

Comment (SebastianPietruszka) [reply src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss]: But I can also pass `ownProps` to circle in Radio component

Comment (SebastianPietruszka) [reply src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss]: Improved

File: src/alto-ui/Icons/Radio.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const Radio = props => (
           {...ownProps}
           d="M18,3A15,15,0,1,0,33,18,15,15,0,0,0,18,3Zm-.47,28.12C10.55,31.12,4.88,25,4.88,18S10.55,4.88,17.53,4.88,31.12,11,31.12,18,24.51,31.12,17.53,31.12Z"
         />
-        <circle cx="18" cy="18" r="6.85" />
+        <circle {...ownProps} cx="18" cy="18" r="6.85" />
```
Comment (renaudtertrais) [inline src/alto-ui/Icons/Radio.js]: Perfect catch!

**Pull request merged**